lld/ELF/InputFiles.cpp

```diff
@@ -655,6 +721,13 @@ template <class ELFT> void ObjFile<ELFT>::parse(bool ignoreComdats) {
     }
   }
 
+  if (hasAArch64BuildAttributes) {
+    // Handle AArch64 Build Attributes and GNU properties:
```

Comment:
Move the comment outside the `if` and then delete the braces (single-line simple statement doesn't need braces)